src/requests/completions.jl

```diff
@@ -414,6 +421,11 @@ function import_completions(doc, offset, rng, ppt, pt, t, is_at_end ,x, CIs, ser
                     end
                 end
             else
+                for (n,doc1) in server.workspacepackages
+                    if startswith(n, t.val) && StaticLint.has_workspace_package(server, n)
```

Comment:
I think this is where we need to go via the env info: Say the code has `using Foo`, then we need to look in the env what UUID `Foo` corresponds to, then we look in the `server.workspacepackages` that is indexed by UUID whether there is a package with that UUID. But we probably shouldn't match by name, because the names don't have to be the same if the package is loaded via an env (and not `LOAD_PATH`).